test/dataset/test_rolling.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+This example shows how to fit a model and evaluate its predictions.
+"""
+
+from math import floor
+import pandas as pd
+from gluonts.dataset.artificial import constant_dataset
+from gluonts.dataset.common import ListDataset
+from gluonts.dataset.rolling_dataset import (
+    StepStrategy,
+    generate_rolling_datasets,
+)
+import pytest
+import itertools
```

Comment:
There is an ordering for imports. See https://www.python.org/dev/peps/pep-0008/#imports